internal/uvm/virtual_device.go

```diff
@@ -16,8 +15,14 @@ const (
 	VPCILocationPathIDType  = "vpci-location-path"
 	VPCIClassGUIDTypeLegacy = "class"
 	VPCIClassGUIDType       = "vpci-class-guid"
+	VPCIDeviceIDTypeLegacy  = "vpci"
+	VPCIDeviceIDType        = "vpci-instance-id"
 )
 
+// this is the well known channel type GUID defined by VMBUS for all assigned devices
+const vmbusChannelTypeGUIDFormatted = "{44c4f61d-4444-4400-9d52-802e27ede19f}"
```

Comment:
I prefer keeping this as just the GUID without braces around it and then add braces at the actual usage location (`GetAssignedDeviceVMBUSInstanceID` in this case). So that if we need to use this GUID somewhere else it is easier.